pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -16,38 +16,8 @@ limitations under the License.
 
 package dockertools
 
```

Comment:
There will be more PRs to clean up the package. Can't really pack them all in one PR.